### PR TITLE
docs: Add admin mode blog

### PR DESCRIPTION
**Pull request by adela-bytebase**

### CODE REVIEW

File: content/blog/change-schema-sql-editor-admin-mode-vs-readonly-mode.md [hunk 1]
```diff
@@ -0,0 +1,131 @@
+---
+title: "Change Schema via SQL Editor Admin Mode vs Read-only Mode"
+author: Ningjing
+published_at: 2022/12/14 16:30:17
+feature_image: /static/blog/change-schema-sql-editor-admin-mode-vs-readonly-mode/2-ways.webp
+tags: How-To
+description: This article will give you step-by-step guidance for SQL Editor, to change schema using Admin Mode and Read-only Mode. 5 mins is more than enough.
```
Comment (tianzhou) [inline content/blog/change-schema-sql-editor-admin-mode-vs-readonly-mode.md:7]: can make the description more terse.

Something like "Step-by-step guide for using SQL Editor in read-only and admin mode"

File: content/blog/change-schema-sql-editor-admin-mode-vs-readonly-mode.md [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+4. Create an instance with the following variables and `root`/`testpwd1` as username/password.
+![add-instance](/static/blog/change-schema-sql-editor-admin-mode-vs-readonly-mode/add-instance.webp)
+
+5. Create a project **test-editor**.
+![project](/static/blog/change-schema-sql-editor-admin-mode-vs-readonly-mode/project.webp)
+
+6. Create a database **test-editor**.
+![create-database](/static/blog/change-schema-sql-editor-admin-mode-vs-readonly-mode/create-database.webp)
+
+After resolve the auto-generated issue, database **db-test** is created successfully.
```
Comment (tianzhou) [inline content/blog/change-schema-sql-editor-admin-mode-vs-readonly-mode.md:61]:
```suggestion
After resolving the auto-generated issue, database **db-test** is created successfully.
```

**Review by tianzhou — CHANGES_REQUESTED** [review]

File: content/blog/change-schema-sql-editor-admin-mode-vs-readonly-mode.md [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+Bytebase now covers all human-to-database scenarios. On one hand, Bytebase can detect risky statements, significantly reducing the potential mistakes made by developers. On the other hand, DBAs can configure the automatic SQL review rules and do manual SQL auditing through Bytebase.
+
+![_](/static/blog/change-schema-sql-editor-admin-mode-vs-readonly-mode/overall.webp)
+For Read-Only Mode, the main scenario is that a developer executes SELECT queries. However, the purpose of this tutorial is to demonstrate the difference between Read-only and Admin modes, that’s why we take CREATE TABLE (DDL) for example.
+
+Make sure you have docker installed and started.
+
+## Step 1 - Setup the environment
+
+1. In the terminal, run the following command to start a Bytebase:
```
Comment (milasuperstar) [inline content/blog/change-schema-sql-editor-admin-mode-vs-readonly-mode.md:23]:
```suggestion
1. In the terminal, run the following command to start Bytebase:
```

File: content/blog/change-schema-sql-editor-admin-mode-vs-readonly-mode.md [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+
+```bash
+docker run --name mysqldtest \
+--publish 3307:3306 \
+-e MYSQL_ROOT_HOST=172.17.0.1 \
+-e MYSQL_ROOT_PASSWORD=testpwd1 \
+mysql/mysql-server:8.0
+```
+
+3. Open [http://localhost:5678/](http://localhost:5678/) in your browser, if you are the first to register, you’ll become **Workspace Owner** by default
```
Comment (milasuperstar) [inline content/blog/change-schema-sql-editor-admin-mode-vs-readonly-mode.md:49]:
```suggestion
3. Open [http://localhost:5678/](http://localhost:5678/) in your browser, if it’s the first time you register, you’ll become **Workspace Owner** by default.
```

File: content/blog/change-schema-sql-editor-admin-mode-vs-readonly-mode.md [hunk 5]
```diff
@@ -0,0 +1,131 @@
...
+3. Click **Run**. The query will execute directly. There is no need for issue creation or approval.
+
+![editor-admin-t2](/static/blog/change-schema-sql-editor-admin-mode-vs-readonly-mode/editor-admin-t2.webp)
+
+- As we encountered in the Read-only section, if you has a properly role, even you type the CREATE TABLE query in the Read-only Mode. You’ll be provided with the option to switch to Admin mode.
+
+![popup-switch](/static/blog/change-schema-sql-editor-admin-mode-vs-readonly-mode/popup-switch.webp)
+
+
+Thus far, you’ve tried two ways in Bytebase to handle TABLE CREATE queries for DBA (Admin Mode) and developer (Readonly) respectively. Next, you can continue exploring SQL Editor modes in Bytebase, maybe with your databases and use cases. Feel free to give us your suggestions!
```
Comment (milasuperstar) [inline content/blog/change-schema-sql-editor-admin-mode-vs-readonly-mode.md:131]:
```suggestion
Thus far, you’ve tried two ways in Bytebase to handle CREATE TABLE queries for DBA (Admin Mode) and developer (Read-only) respectively. Next, you can continue exploring SQL Editor modes in Bytebase, maybe with your own databases and use cases. Feel free to give us your suggestions!
```

**Review by Candybase — APPROVED** [review]
LGTM

**Pull request closed without merging**